### ISSUE TITLE
When multiple vehicles are around missle and bomb sounds do not trigger

### ISSUE THREAD

**Issue by Jake762x51** [PapaJoesSoup/BDArmory#209]
When you want to fight against AI enemies or just are in a scene with multiple operating vehicles every-time you fire away a sidewinder or the like you do not hear any sound at all !

Please fix this, i would love to see this fixed.

I love this mod so much, thank you for doing what you're doing ! 

KSP 1.3
This problem occurs even if you use a clean install and just have BDArmory installed.

**Comment by gomker** [comment]
Hello,

Can you let me know if this issue exists when using the reference test vessels. You can find a copy in `\GameData\BDArmory\craft`  or [here]( https://github.com/PapaJoesSoup/BDArmory/tree/master/BahaTurret/Distribution/GameData/BDArmory/craft)

Please use the `BDA - TestPlaneAI - v3.craft` and let us know if the issue exists with just the sidewinder or other missiles. Our testers have not reported an issue like this before.

Regards,

**Comment by Jake762x51** [comment]
Yes, i just tried a battle between 4 of these premade aircraft and the missile sounds and bomb launching sounds do not get played. If you want i can send you a video i recorded of it as proof.

**Comment by Jake762x51** [comment]
Can you please further investigate these audio issues, i continue having them even after making sure i have a clean KSP install and the newest BDA and other necessary mods.

Newest Version of KSP
Newest Version of BDA
Newest Version of Windows 10 Pro 64Bit

Log is attached 
[KSP.txt](https://github.com/PapaJoesSoup/BDArmory/files/1132274/KSP.txt)
